text-encoding/text-encoding.d.ts

```diff
@@ -32,14 +32,29 @@ declare namespace TextEncoding {
     interface TextEncodeOptions {
         stream?: boolean;
     }
+
+    interface TextEncoderStatic {
```

Comment:
consider using the [ghost module](http://definitelytyped.org/guides/best-practices.html#ghost-modules) pattern.